CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+>  
+Attribution: This Contributor Agreement is adapted from the node.js project available here: https://github.com/nodejs/node/blob/main/CONTRIBUTING.md 
+
+## [How to Create Good Issues or Pull Request](howto)
+ 
+Content to be added. 
+
+## [License](license)
+
+The TIDES specification is licensed under the Apache License 2.0 as defined in [LICENSE][license-file] file.
+
+## [Project Governance](governance)
+ 
+Ahead of Version 1.0 release, the governance for the TIDES specification is being kept limited and lightweight. The governance approach will be revisited as release of Version 1.0 approaches.
+
+Development of the TIDES specification shall be managed by the following groups:
+* Leadership
+* Product Management Team
+* Registered Contributors
+* All Other Stakeholders
+
+These groups will have the following rights and responsibilities:
+
+### Leadership 
+
+Leadership is responsible for overall direction and decision-making on the project including:
+* approval of registered contributors
+* creation, scoping, and management of working groups, 
+* approval of the final specification for Version 1.0
+* approval of changes to project governance
+
+Leadership Group Members
+* John Levin, Metro Transit (Minneapolis-St. Paul, MN)
+
+Leadership Group Member [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Admin
+ 
+### Product Management Team (PMT)
+
+The Product Management Team (PMT) is responsible for creating and maintaining backbone standards infrastructure, processes, and resources to support the development of the TIDES specification.  The PMT will support Leadership in developing, reviewing, and recommending for approval changes to the draft specification.  The PMT will support Contributors and Stakeholders in their work on the specification.  
+
+PMT Group Members
+* Hunter Owens, Caltrans
+* Jameelah Young, Jarv.us (on behalf of Caltrans)
+* Elizabeth Sall, UrbanLabs LLC (on behalf of Caltrans)
+* Benjamin Bressette, Caltrans
+* Blake Fleisher, Jarv.us (on behalf of Caltrans)
+* Joey Reid, Metro Transit (Minneapolis-St. Paul, MN)
+
+PMT Group Member [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Admin
+ 
+### Contributors
+
+Contributors actively work to develop the TIDES specification. They propose additions, modifications, and improvements to the speciation document through issues and pull requests in this GitHub repository.
+
+Contributors must request [here][contributor-registration] to be registered in order to gain access.  Requests to become a Contributor must be approved by project Leadership.
+ 
+Contributor Group Members: 
+The list of registered contributors is maintained in the [contributors.md][contributors-file] file.
```

Comment:
This is an invalid link, should be something like, `[contributors.md](CONTRIBUTORS.md)`